test/e2e/kic/kubernetes/kong_ingress.go

```diff
@@ -37,8 +37,8 @@ func testEchoServer(port int) error {
 }
 
 func KICKubernetes() {
-	if IsApiV2() {
-		fmt.Println("Test not supported on API v2")
+	if IsApiV2() || IsIPv6() {
```

Comment:
Is there a github issue link we could leave to give more context on the IPv6 issue?

Reply:
Do you mean link in a comment here in the code? Here is one link  - https://github.com/Kong/kubernetes-ingress-controller/issues/1017

Reply:
Yup, in the code would be ideal.